src/components/general/FilterList.js

```diff
@@ -6,15 +6,26 @@ import Ionicons from "react-native-vector-icons/Ionicons";
 import { Button } from "./Button";
 import { ButtonLayout } from "./ButtonLayout";
 
-/*
-	FilterList
-
-	Documentation in progress
+const dimension = Dimensions.get("window");
 
-	Full Examples
+/**
+	FilterList
+	@class
+
+	@param {(String[] | Object[])} data		An array of the data that will be displayed.
```

Comment:
Use spaces for all identation after the initial tab and make all descriptions of params aligned.